server/bin/pbench-base.sh

```diff
@@ -145,9 +145,9 @@ function quarantine () {
 
 function log_info {
     if [[ -z "${2}" ]]; then
-        printf -- "%s\n" "${1}"
+        printf -- "%b\n" "${1}"
     else
-        printf -- "%s\n" "${1}" | tee -a "${2}"
+        printf -- "%b\n" "${1}" | tee -a "${2}"
```

Comment:
We should probably use '%b' in `log_error` as well. At some point, somebody is going to use '\n' in an error message.

Reply:
ok, sure